ocs_ci/ocs/resources/noobaa.py

```diff
@@ -48,16 +48,22 @@ def __init__(self):
             aws_secret_access_key=self.access_key
         )
 
-    def s3_create_bucket(self, bucketname):
+    def s3_create_bucket(self, bucketname, region='us-east-1'):
         """
         Args:
             bucketname: Name of the bucket to be created
+            region: Name of the region for the bucket to be created on (defualt is us-east-1, as the SDK default)
 
         Returns:
             s3.Bucket object
 
         """
-        return self.s3_resource.create_bucket(Bucket=bucketname)
+        return self.s3_resource.create_bucket(
+            Bucket=bucketname,
+            CreateBucketConfiguration={
```

Comment:
Can we pass multiple regions here? Seems like 'LocationConstraint' key can get only a string accroding to https://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Types/CreateBucketConfiguration.html

Reply:
Sorry, this one:
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-example-creating-buckets.html

Reply:
A bucket is always associated with 1 region only.

Reply:
It's clear now, after an offline discussion.
Thanks @nimrod-becker